lib/payload.rb

```diff
@@ -36,7 +36,7 @@ def record
 
   def model_class_for(model_class_name)
     candidate_class = model_class_name.classify.safe_constantize
-    unless candidate_class && (candidate_class < ApplicationRecord)
+    unless candidate_class.respond_to?(:create_or_update_from_json)
       raise(InvalidMessage, "Unrecognized model: #{model_class_name}")
```

Comment:
We currently have lots of deadletters from barcodes, which get routed to the WH, but are handled elsewhere. This lets the WH just ignore the messages.